pkgs/applications/window-managers/cagebreak/default.nix

```diff
@@ -3,25 +3,29 @@
 , wlroots, wayland-protocols, pixman, libxkbcommon
 , cairo , pango, fontconfig, pandoc, systemd
 , withXwayland ? true, xwayland
-, nixosTests
+, nixosTests, libdrm
 }:
 
 stdenv.mkDerivation rec {
   pname = "cagebreak";
-  version = "1.6.0";
+  version = "1.7.0";
 
   src = fetchFromGitHub {
     owner = "project-repo";
     repo = "cagebreak";
     rev = version;
-    hash = "sha256-F7fqDVbJS6pVgmj6C1/l9PAaz5yzcYpaq6oc6a6v/Qk=";
+    sha256 = "10ps59dbvxalkdy0rs67k4dgkhy16xspdwpxl551dcxrgcj2740y";
```

Comment:
I thought there was a general move towards replacing `sha256` with the new hashes from Nix 2.4?

Reply:
I didn't get the memo on that one, source?

Reply:
I don't have any link. I came to this understanding from discussions in \#nixos on IRC.

Anyhow, the new hashes are only generated when using nix 2.4 tooling IIRC (`nixUnstable`) so it doesn't make sense to move to something that has not been stabilized yet.

Reply:
Right now it does not matter which hash you are using. Just use the one that your nix-daemon provides.

> Anyhow, the new hashes are only generated when using nix 2.4 tooling IIRC (nixUnstable) so it doesn't make sense to move to something that has not been stabilized yet.


If you are using a newer nix-daemon then you would need to convert it every time which is tedious and not required.